src/handlers/event/delete_on_reaction.rs

```diff
@@ -14,10 +14,7 @@ pub async fn handle(ctx: &Context, reaction: &Reaction) -> Result<()> {
 		.wrap_err("Couldn't fetch message from reaction!")?;
 
 	if let Some(interaction) = &message.interaction {
-		if interaction.kind == InteractionType::Command
```

Comment:
I removed this because I don't think you can have any other interaction kind on a message

Reply:
this is enum is marked as non-exhaustive and therefore *could* in the future. this is a purposeful guard as we should only be deleting messages from commands anyways

Reply:
Oh I guess starting an activity might be a different type and you wouldn't want people to hide that